src/AzureStack/Admin/InfrastructureInsightsAdmin/InfrastructureInsights.Admin/Microsoft.AzureStack.Management.InfrastructureInsights.Admin.csproj

```diff
@@ -7,14 +7,15 @@
     <PropertyGroup>
         <PackageId>Microsoft.AzureStack.Management.InfrastructureInsights.Admin</PackageId>
         <Description>Provides developers with libraries for the health admin under AzureStack to manage health alerts.</Description>
-        <VersionPrefix>0.2.0-preview</VersionPrefix>
+        <VersionPrefix>0.3.0-preview</VersionPrefix>
         <AssemblyName>Microsoft.AzureStack.Management.InfrastructureInsights.Admin</AssemblyName>
         <PackageTags>Microsoft Azure Stack;InfrastructureInsights;REST HTTP client;azureostackfficial;netcore451511</PackageTags>
         <PackageReleaseNotes>
             <![CDATA[
                 Preview release of the InfrastructureInsights administrator SDK which allows administrators  
                 1) View the health of their AzureStack stamp resources 
                 2) View and manage alerts
+                Repair API is supported in azurestack 1811+
```

Comment:
nit: please stick to numbering scheme

Reply:
@dsgouda Should we only put in new things for release notes?

Reply:
Ideally yes, that way users know what changed from the last version

Reply:
Just updated the PR.